src/dataset-builder/ConceptSearch.ts

```diff
@@ -109,15 +108,19 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
               { width: 100, key: 'hierarchy' },
             ],
             rows: _.map((concept) => {
-              const [label, iconName]: [string, IconId] = _.contains(concept, cart)
-                ? ['remove', 'minus-circle-red']
-                : ['add', 'plus-circle-filled'];
+              const checked = _.contains(concept, cart);
               return {
                 name: div({ style: { display: 'flex' } }, [
-                  h(Link, { 'aria-label': `${label} ${concept.id}`, onClick: () => setCart(_.xor(cart, [concept])) }, [
-                    icon(iconName, { size: 16 }),
-                  ]),
-                  div({ style: { marginLeft: 5 } }, [
+                  h(
+                    LabeledCheckbox,
+                    {
+                      'aria-label': checked ? `uncheck ${concept.id}` : `check ${concept.id}`,
```

Comment:
if the box is checked, the action item would be to uncheck it and vice versus. if the box is not checked, the action is checking it.